### PR TITLE
Implement Fallback to User Balance for Transactions When Relayer Fails

### PR DESCRIPTION
**Pull request by Pessina**
This PR introduces a fallback mechanism for transactions when the relayer service fails to relay them due to specific issues such as actions not being allowed or the sender/receiver not being whitelisted. In such cases, the transaction will be retried using the user's own balance, ensuring that the transaction can still be processed without dependency on the relayer's current state or configuration. This enhancement improves the robustness of transaction handling by providing an alternative path for transaction completion.

### CODE REVIEW

File: src/lib/near-fastauth-wallet.ts [hunk 1]
```diff
@@ -152,14 +156,119 @@ const FastAuthWallet: WalletBehaviourFactory<
...
+
+  const _signAndSend = async ({
+    transactions,
+    callbackUrl,
+  }: {
+    transactions: Optional<Transaction, 'signerId'>[];
+    callbackUrl?: string;
+  }): Promise<void> => {
+    const account = _state.wallet.account();
+    const connectedNearAccountWallet = _state.wallet._connectedAccount;
```
Comment (esaminu) [inline src/lib/near-fastauth-wallet.ts]: These two are the same yes? Why not just use account?

Comment (Pessina) [reply src/lib/near-fastauth-wallet.ts]: Idd, they are the same. Updated to use only one

File: src/lib/fastAuthWalletConnection.ts [hunk 2]
```diff
@@ -302,30 +322,81 @@ export class FastAuthWalletConnection {
     );
     newUrl.searchParams.set('success_url', callbackUrl || currentUrl.href);
     newUrl.searchParams.set('failure_url', callbackUrl || currentUrl.href);
```
Comment (esaminu) [inline src/lib/fastAuthWalletConnection.ts:324]: Very nice but with time these params can be removed for simplicity so that we have iframe only logic 👍

Comment (Pessina) [reply src/lib/fastAuthWalletConnection.ts:324]: Yeah, let's open a ticket for this clean up! Will be amazing to remove the duplicated logic

**Pull request merged**